docs/articles/new-expensify/expensify-card/Enable-Expensify-Card-notifications.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Enable Expensify Card notifications
+description: Allow notifications from Expensify
+---
+<div id="new-expensify" markdown="1">
+
+The Expensify mobile app sends you real-time notifications for spending activity on your Expensify Visa® Commercial Card, including
+- Purchase notifications, including declined payments
+- Fraudulent activity alerts
+- Requests for purchases that require a SmartScanned receipt 
+
+There are two steps to enable Expensify Card notifications. You’ll first enable alerts on your workspace, then you’ll enable notifications on your device.
+
+# Step 1: Enable alerts on your workspace
+
+{% include selector.html values="desktop, mobile" %}
+
+{% include option.html value="desktop" %}
+1. From your Expensify Chat inbox, click the dropdown on the logo or avatar that is in the top left corner. 
+2. Select the workspace you want to update the notification settings for.
+3. Click the workspace chat in your inbox (it will be the chat that has your workspace’s name as the chat title).
+4. Click the header at the top of the chat.
+5. Click **Settings**.
+6. Click **Notify me about new messages** and select **Immediately**.
+{% include end-option.html %}
+
+{% include option.html value="mobile" %}
+1. From your Expensify Chat inbox, tap the dropdown on the logo or avatar that is in the top left corner. 
+2. Select the workspace you want to update the notification settings for.
+3. Tap the workspace chat in your inbox (it will be the chat that has your workspace’s name as the chat title).
+4. Tap the header at the top of the chat.
+5. Tap **Settings**.
+6. Tap **Notify me about new messages** and select **Immediately**.
+{% include end-option.html %}
+
+{% include end-selector.html %}
+
+# Step 2: Enable notifications on your device
+
+{% include selector.html values="desktop, mobile" %}
```

Comment:
@ren-jones we don't need a platform selector here. We can just show the info directly

```suggestion

```